liquidctl/driver/nvidia.py

```diff
@@ -0,0 +1,171 @@
+"""liquidctl drivers for NVIDIA graphics cards.
+
+Copyright (C) 2020–2020  Jonas Malaco and contributors
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+from enum import Enum, unique
+import logging
+import re
+
+from liquidctl.driver.smbus import SmbusDriver
+from liquidctl.error import NotSupportedByDevice
+from liquidctl.util import check_unsafe
+
+_LOGGER = logging.getLogger(__name__)
+
+_NVIDIA = 0x10de
+_EVGA = 0x3842
+
+_NVIDIA_GTX_1080 = 0x1b80
+
+
+class EvgaPascal(SmbusDriver):
+    """NVIDIA series 10 (Pascal) graphics card from EVGA."""
+
+    ADDRESS = 0x49
+    REG_MODE = 0xc
+    REG_RED = 0x9
+    REG_GREEN = 0xa
+    REG_BLUE = 0xb
+    REG_PERSIST = 0x23
+    PERSIST = 0xe5
+
+    @unique
+    class Mode(bytes, Enum):
+        def __new__(cls, value, required_colors):
+            obj = bytes.__new__(cls, [value])
+            obj._value_ = value
+            obj.required_colors = required_colors
+            return obj
+
+        OFF = (0b000, 0)
+        FIXED = (0b001, 1)
+        RAINBOW = (0b010, 0)
+        BREATHING = (0b101, 1)
+
+        def __str__(self):
+            return self.name.capitalize()
+
+    @classmethod
+    def probe(cls, smbus, vendor=None, product=None, address=None, match=None,
+              release=None, serial=None, **kwargs):
+        GTX_1080_FTW = 0x6286
```

Comment:
How come this value is here and not at the top with the other constants?

Reply:
I guess it's a personal preference: the value is not needed outside of this function, so I decided to keep it local.